selftests/functional/test_output.py

```diff
@@ -272,8 +272,8 @@ def test_output_compatible_setup_3(self):
         output = result.stdout + result.stderr
         expected_rc = exit_codes.AVOCADO_ALL_OK
         tmpdir_contents = os.listdir(tmpdir)
-        self.assertEqual(len(tmpdir_contents), 4,
-                         'Not all resources dir were created: %s' % tmpdir_contents)
+        self.assertEqual(len(tmpdir_contents), 1, 'Not all resources dir were'
```

Comment:
This check doesn't involve resources dirs anymore.  The only thing you'll find in `tmpdir` is the HTML result file itself (with a `tmp?????` like name).

Reply:
I know, I kept the check as it can uncover issues where html plugin produces multiple (eg. temporary) files. Anyway I can change it to the same check as json and xunit does, but I think it's good to check no other files were produced. (although the message is a bit misleading)

Reply:
No problem in checking that only the HTML file was created.  IMO, it's a bit of a strange check to have at this exact context, but, it's up to you.  If you keep the check, please change the message.